test/junit/scala/tools/nsc/QuickfixTest.scala

```diff
@@ -6,25 +6,30 @@ import org.junit.Test
 import java.nio.file.Files
 import scala.reflect.internal.util.BatchSourceFile
 import scala.reflect.io.AbstractFile
+import scala.tools.nsc.reporters.StoreReporter
 import scala.tools.testkit.BytecodeTesting
 import scala.tools.testkit.ReleasablePath._
 import scala.util.Using
 
-import reporters.StoreReporter
-
 class QuickfixTest extends BytecodeTesting {
-  def testQuickfix(a: String, b: String, args: String, checkInfo: StoreReporter.Info => Boolean = _ => true): Unit = if (!scala.util.Properties.isWin) {
-    Using.resource(Files.createTempFile("unitSource", "scala")) { src =>
-      Files.write(src, a.getBytes)
-      val c = BytecodeTesting.newCompiler(extraArgs = args)
-      val r = c.newRun()
-      val f = AbstractFile.getFile(src.toFile.getAbsolutePath)
-      r.compileSources(List(new BatchSourceFile(f)))
-      assertEquals(b, new String(Files.readAllBytes(src)))
-      for (info <- c.global.reporter.asInstanceOf[StoreReporter].infos)
-        assert(checkInfo(info), info)
+
+  def testQuickfixs(as: List[String], b: String, args: String, checkInfo: StoreReporter.Info => Boolean = _ => true): Unit =
```

Comment:
I didn't want to provoke you by indenting the `if`, but glad you did that and added multiple sources.